lib/membrane_udp/common_socket_behaviour.ex

```diff
@@ -13,7 +13,7 @@ defmodule Membrane.UDP.CommonSocketBehaviour do
         ) ::
           {[Membrane.Element.Action.common_actions() | Membrane.Element.Action.setup()],
            Membrane.Element.state()}
-  def handle_setup(ctx, %{local_socket: %Socket{} = local_socket} = state) do
+  def handle_setup(ctx, %{local_socket: %Socket{socket_handle: nil} = local_socket} = state) do
```

Comment:
[NIT] Could you fetch this `local_socket` as `state.local_socket`  in the implementation below? This pattern match has become crazily long IMO :D